### PR TITLE
CORDA-1141: Kryo client serilization scheme should use AMQP for P2P c…

### PR DESCRIPTION
**Pull request by adagys**
…ontext.

### CODE REVIEW

**Comment by rick-r3** [comment]
@fenryka Is going to finish this off or raise a replacement, since we need to deal with the potential for custom serialiser registration on the RPC client side, and it's not there yet.

**Comment by mikehearn** [comment]
OK. Can you explain this in a bit more detail in a comment please? Because the change just looks wrong, doesn't it? I'm sure it's right but it's not clear to me why the Kryo part is using "AMQP_CONTEXT". I hope after we finish this work and Kryo is hiding in the corner doing just checkpointing we can simplify this code a lot, because it's quite confusing.

File: client/rpc/src/main/kotlin/net/corda/client/rpc/internal/KryoClientSerializationScheme.kt [hunk 1]
```diff
@@ -43,7 +43,7 @@ class KryoClientSerializationScheme : AbstractKryoSerializationScheme() {
                         registerScheme(KryoClientSerializationScheme())
                         registerScheme(AMQPClientSerializationScheme(emptyList()))
                     },
-                    KRYO_P2P_CONTEXT,
+                    AMQP_P2P_CONTEXT,
```
Comment (fenryka) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/KryoClientSerializationScheme.kt]: Alas it isn't this simple

**Review by fenryka — CHANGES_REQUESTED** [review]
I'll take this over, as discussed, the problem is access to pluggable serialisers for the library, which is tricky given how RPC is enabled all over the place

**Pull request closed without merging**